src/qt/bitcoingui.cpp

```diff
@@ -672,7 +672,7 @@ void BitcoinGUI::setNumConnections(int count)
     labelConnectionsIcon->setToolTip(tr("%n active connection(s) to Bitcoin network", "", count));
 }
 
-void BitcoinGUI::setNumBlocks(int count, const QDateTime& blockDate)
+void BitcoinGUI::setNumBlocks(int count, const QDateTime& blockDate, const CBlockIndex *tip)
```

Comment:
Hmm... but i don't want to execute `getVerificationProgress()` during the synchronous core signal. I guess it's better to call this function over the UI thread.
But let me have a closer look...